src/index.html

```diff
@@ -15,8 +15,91 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="index.html"
+        class="logo-link"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Moyo logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav-links">
+          <li class="nav-link-item">
+            <a
+              class="nav-link is-active"
+              href="/apple"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav-link-item">
+            <a
+              class="nav-link"
+              href="/samsung"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav-link-item">
+            <a
+              class="nav-link"
+              href="/smartphones"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="nav-link-item">
+            <a
+              class="nav-link"
+              href="/laptops"
+              data-qa="hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav-link-item">
+            <a
+              class="nav-link"
+              href="/gadgets"
```

Comment:
Don't use spaces in `<a>` tag's `href` property. Anchor links start with the `#` symbol if they are meant to be internal page anchors.